distribution/openrct2.d.ts

```diff
@@ -185,6 +185,19 @@ declare global {
          */
         sharedStorage: Configuration;
 
+        /**
+         * Gets the storage for the current plugin or another plugin. Data is persisted
+         * for the current loaded park, and is stored inside the .park file. Any references
```

Comment:
```suggestion
         * Gets the storage for the current plugin if no name was specified or for the plugin of given name. Data is persisted
```
Makes this a bit more clear that this parameter should be mostly optional, at first thought it would seem like I have to pass my own plugin name there. You can probably lay out that sentence better than I can